common/maplib/map.mjs

```diff
@@ -234,13 +235,33 @@ class GameMap {
     }
     return ret;
   }
-}
 
+  /**
+   * Get the Spawn Points from all maps.
+   * @return {mapCoord Array} : Random spawn points with mapCoord type
```

Comment:
No, this function should return a single spawn point.
`@return {MapCoord} A random spawn point`